crime-applications-adaptor/src/test/java/uk/gov/justice/laa/crime/applications/adaptor/config/MockServicesConfiguration.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.laa.crime.applications.adaptor.config;
+
+public class MockServicesConfiguration {
+
+    public static final String CLIENT_SECRET = "TESTVjBtRXlpQGNDJGZIKU5MQUVtd2NPY0FbLVN6JGg=";
```

Comment:
Can probably make these constants private

Reply:
Updated to private